packages/browser-wallet/src/popup/pages/Account/Tokens/Tokens.tsx

```diff
@@ -97,9 +97,16 @@ function Collectibles({ account, toDetails }: ListProps) {
                     <div className="token-list__unique-name">
                         {token.metadata.name}
                         <AtomValue atom={contractBalancesFamily(account.address, token.contractIndex)}>
-                            {({ [token.id]: b }) =>
-                                b === 0n && <div className="token-list__not-owned text-faded">{t('unownedUnique')}</div>
-                            }
+                            {({ [token.id]: b }) => (
+                                <>
+                                    {b === 0n && <div className="token-list__ownership">{t('unownedUnique')}</div>}
+                                    {b > 1n && !token.metadata.decimals && (
```

Comment:
Why do you only show the balance if it does not specify decimals?

Reply:
Yes, doesn't make any sense.. you know me and my boolean flags.. not friends..

Reply:
Wouldn't it make sense to also do the `balance / BigInt(10 ** decimals) !== 1n` here?

Reply:
Because currently if I have 2 decimals and 100 (Which should be interpreted as 1), the value is displayed.

Reply:
Yes, would make sense